torch/_export/serde/schema.py

```diff
@@ -27,6 +27,8 @@ class ScalarType(IntEnum):
     COMPLEXDOUBLE = 11
     BOOL = 12
     BFLOAT16 = 13
+    FLOAT8_E4M3FN = 14
+    FLOAT8_E5M2 = 15
```

Comment:
could we follow this enumeration? https://github.com/pytorch/pytorch/blob/b740a1b96cac9d1ca65ffaf59e66185306e8afd0/c10/core/ScalarType.h#L85-L88

Reply:
Hi Angela, I have made modifications accordingly.